discussions.go

```diff
@@ -0,0 +1,477 @@
+//
+// Copyright 2017, Sander van Harmelen
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package gitlab
+
+import (
+	"fmt"
+	"net/url"
+	"time"
+)
+
+// DiscussionsService handles communication with the discussions related methods
+// of the GitLab API.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/discussions.html
+type DiscussionsService struct {
+	client *Client
+}
+
+// Discussion represents a GitLab discussion.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/discussions.html
+type Discussion struct {
+	ID             string `json:"id"`
+	IndividualNote bool   `json:"individual_note"`
+	Notes          []Note `json:"notes"`
+}
+
+func (n Discussion) String() string {
+	return Stringify(n)
+}
+
+// ListIssueDiscussionsOptions represents the available ListIssueDiscussions() options.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/discussions.html#list-project-issue-discussions
+type ListIssueDiscussionsOptions struct {
+	ListOptions
+	OrderBy *string `url:"order_by,omitempty" json:"order_by,omitempty"`
+	Sort    *string `url:"sort,omitempty" json:"sort,omitempty"`
```

Comment:
Are you sure these options are supported here? As the docs don't mention them.

Reply:
No, they are not, and you are correct. I will remove them and make sure to check with the API docs for the remaining `*Options` structs.